packages/onboarding/src/step-container/style.scss

```diff
@@ -35,6 +35,7 @@
 		@include break-small {
 			margin-top: 25vh;
 			display: flex;
+			flex-direction: row;
```

Comment:
We might need to add it to https://github.com/Automattic/wp-calypso/blob/73f9ef1d35273fbfceedc535a5a555a752ac22e3/client/landing/stepper/declarative-flow/internals/global.scss#L157

For example,
```css
&.is-horizontal-layout {
  flex-direction: row;
}
```